app/src/main/java/io/github/zadam/triliumsender/CustomTrustClient.kt

```diff
@@ -0,0 +1,157 @@
+package io.github.zadam.triliumsender
+
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.tls.HandshakeCertificates
+import java.io.ByteArrayInputStream
+import java.io.IOException
+import java.io.InputStream
+import java.security.KeyManagementException
+import java.security.NoSuchAlgorithmException
+import java.security.SecureRandom
+import java.security.cert.CertificateException
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+class CustomTrustClient {
```

Comment:
This class looked much prettier in Java. I don't know Kotlin, but I guessed you wouldn't want to mix languages.

Reply:
My original code: https://gist.github.com/jkurei/26eeba6cbdc3f4c16394d9cbe0d3acfd